core/src/jvmMain/kotlin/de/p7s1/qa/sevenfacette/db/SqlStatement.kt

```diff
@@ -0,0 +1,62 @@
+package de.p7s1.qa.sevenfacette.db
+
+/**
+ * Represents a prepared database statement
+ *
+ * @param [statement] is the original statement with placeholder
+ * @param [args] are the arguments with the placeholder should be replaced
+ *
+ * @author Patrick Döring
+ */
+class SqlStatement(statement: String, vararg args: Any?) {
+    val origStatement: String = statement
+    var sqlStatement: String = statement
+    var arguments: Any = args
+    var regex = "(?<!^')\\?(?!.*?')"
+
+    init {
+        if (args.isNotEmpty()) {
+            replaceAll(*args as Array<out Any>)
+        }
+    }
+
+    /**
+     * Validates if the statement contains a placeholder
+     *
+     * @return [true] or [false]
+     */
+    fun validate() : Boolean{
+        return !sqlStatement.contains(Regex(regex))
```

Comment:
An sich passt der Funktionsname nicht zu dem was tatsächlich passiert. Wäre nicht sowas wie "contaisPlaceholder" besser?

Reply:
In diesem Fall validieren wir ob wir ein echtes prepared Statement haben. Daher validate. 
Wie wäre es mit validatePreparedStatement?